configs/mainnet.yaml

```diff
@@ -146,9 +146,7 @@ MAX_REQUEST_BLOCKS_DENEB: 128
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # 6 subnets
 BLOB_SIDECAR_SUBNET_COUNT: 6
-# 6 blobs
-MAX_BLOBS_PER_BLOCK: 6
-# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK (= 128 * 6) sidecars
+# MAX_REQUEST_BLOCKS_DENEB * get_max_blobs_per_block(DENEB_FORK_EPOCH) (= 128 * 6) sidecars
 MAX_REQUEST_BLOB_SIDECARS: 768
```

Comment:
This was probably discussed somewhere else, but can we just remove `MAX_REQUEST_BLOB_SIDECARS`?